examples/rfcavity/input_rfcavity_ref_part.in

```diff
@@ -0,0 +1,84 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000  # outside tests, use 1e5 or more
+beam.units = static
+beam.kin_energy = 230
+beam.charge = 1.0e-10
+beam.particle = electron
+beam.distribution = waterbag
+beam.lambdaX = 0.352498964601e-3
+beam.lambdaY = 0.207443478142e-3
+beam.lambdaT = 0.70399950746e-4
+beam.lambdaPx = 5.161852770e-6
+beam.lambdaPy = 9.163582894e-6
+beam.lambdaPt = 0.260528852031e-3
+beam.muxpx = 0.5712386101751441
+beam.muypy = -0.514495755427526
+beam.mutpt = -5.05773e-10
```

Comment:
Can this be removed or does our code structure enforce it right now?
```suggestion
```

Reply:
Yes, this works.

Reply:
OK cool! Yes then let us remove this here?